dart/CMakeLists.txt

```diff
@@ -150,7 +150,11 @@ if(DART_ENABLE_SIMD)
     # in CMake time, then I would gratefully apply these options differently
     # depending on the architectures.
   elseif(CMAKE_COMPILER_IS_GNUCXX)
-    target_compile_options(dart PUBLIC -march=native -faligned-new)
+    set(native_flags "-march=native")
+    if(GCC_VERSION VERSION_GREATER 7.0)
+      set(native_flags "-march=native;-faligned-new")
+    endif()
+    target_compile_options(dart PUBLIC ${native_flags})
```

Comment:
According to the [documentation](https://cmake.org/cmake/help/v3.12/command/target_compile_options.html), repeated calls for the same `<target>` append options in the order called, which I just learned. So I think we can simplify these as:
```cmake
    target_compile_options(dart PUBLIC -march=native)
    if(GCC_VERSION VERSION_GREATER 7.0)
      target_compile_options(dart PUBLIC -faligned-new)
    endif()
```
Also, I don't think we need to enclose the options with double quotes.

Reply:
> According to the [documentation](https://cmake.org/cmake/help/v3.12/command/target_compile_options.html), repeated calls for the same `<target>` append options in the order called, which I just learned.

I didn't know that! I updated the changes..